projects/components/src/table/table.component.ts

```diff
@@ -66,15 +68,19 @@ import { TableColumnConfigExtended, TableService } from './table.service';
       class="table"
     >
       <!-- Columns -->
-      <div *ngFor="let columnDef of this.columnConfigsSubject.value; index as index">
+      <div *ngFor="let columnDef of this.columnConfigs$ | async; trackBy: this.trackItem; index as index">
         <ng-container [cdkColumnDef]="columnDef.id">
           <cdk-header-cell
             *cdkHeaderCellDef
             [style.flex-basis]="columnDef.width"
             [style.max-width]="columnDef.width"
             class="header-cell"
           >
+            <div *ngIf="index !== 0" class="header-column-handle" (mousedown)="this.onResizeMouseDown($event, index)">
```

Comment:
Rather than working by index and in the dom, requerying each time - couldn't we pass in the column def, and have the column def hold the changing data? Or at least serve as a reference to it if we're not mutating the column already.

Reply:
I'm not sure I understand what the goal is here. What does this give us?

Reply:
The goal is preventing multiple DOM queries per mouse move event when those query results are known - they only change if columns are reconfigured.

Reply:
Guess I'm still not following. The mousedown event is only triggered when there is mousedown over the column-handle. It doesn't fire off any other time.

Reply:
Discussed offline - the comment position doesn't make a whole lot of sense here, but the ask is to move the expensive querying and measuring (if possible) to the mouse down handler, rather than in each cycle of the mousemove handler.